src/files.ts

```diff
@@ -134,7 +135,7 @@ export function parseFileIgnores(path: string): string[] {
 export function getGlobPatterns(supportedFiles: SupportedFiles): string[] {
   return [
     ...supportedFiles.extensions.map(e => `*${e}`),
-    ...supportedFiles.configFiles.filter(e => !IGNORE_FILES_NAMES.includes(e)),
+    ...supportedFiles.configFiles.filter(e => !EXCLUDED_NAMES.includes(e)),
```

Comment:
```
export const EXCLUDED_NAMES = [GIT_FILENAME, GITIGNORE_FILENAME, DCIGNORE_FILENAME];
export const IGNORE_FILES_NAMES = [GITIGNORE_FILENAME, DCIGNORE_FILENAME, DOTSNYK_FILENAME];
```

This allows `.snyk` but now excludes `.git`. Interestingly, `.git` seems to already be excluded by [this](https://github.com/snyk/code-client/blob/master/src/constants.ts#L19). Still, there could be some unexpected side-effects?

Reply:
Good catch. We only use `EXCLUDED_NAMES` once in this project (and I can't see it being used by any `code-client` consumers), so I can tweak the object.

Reply:
Done - tested locally, `.git` is indeed excluded elsewhere.